_schedule.yml

```diff
@@ -69,7 +69,7 @@ scheduled-docs:
             - type: Slides
               href: 2-summarizing-data/01-summarizing-categorical-data/slides.qmd
             - type: WS
-              href: 2-summarizing-data/01-summarizing-categorical-data/ps.qmd
+              href: 2-summarizing-data/01-summarizing-categorical-data/ws.qmd
```

Comment:
@lucasadam7 I believe this is what caused that rendering error. In changing the name of problem sets to worksheets, there are a good number of links and filenames that need to be updated. Thanks for flagging this one!